database/entity/0060-add_verified_at_to_communities/Community.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  BaseEntity,
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm'
+
+@Entity('communities')
+export class Community extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ name: 'public_key', type: 'binary', length: 64, default: null, nullable: true })
+  publicKey: Buffer
```

Comment:
```suggestion
  publicKey: Buffer | null
```

Why nullable?

Reply:
Many THX...
nullable is default, now its NOT NULL